api/src/main/resources/org/openmrs/module/pihmalawi/reporting/reports/sql/ic3_register.sql

```diff
@@ -113,9 +113,7 @@ CALL getEncounterDateForObs(5089, @reportEndDate, 'last', 'lastWeightDate');
 CALL getNumericObsBeforeDate(5090, @reportEndDate, 'last', 'lastHeight');
 CALL getNumericObsBeforeDate(5089, @reportEndDate, 'last', 'lastWeight');
 -- Diagnoses Logic
-CALL diagnosesLogic(); -- Must be last!
```

Comment:
See my comment above.  We should add some logic to allow inline comments like this, which can be really helpful for clarity of what are very long and complicated sql files...